github/orgs_projects.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ListProjectsOptions specifies the optional parameters to the
+// OrganizationsService.ListProjects method.
+type ListProjectsOptions struct {
```

Comment:
What do you think of `ProjectListOptions` as an alternative name for this struct?

Also, we should probably update `RepositoriesService.ListProjects` method to also use it, given that https://developer.github.com/v3/projects/#list-repository-projects has the same list parameters as https://developer.github.com/v3/projects/#list-organization-projects. Are you okay with doing that as part of this PR?